src/convert.jl

```diff
@@ -8,6 +8,7 @@
 
 # this covers most conversions and "statically-sized reshapes"
 @inline convert(::Type{SA}, sa::StaticArray) where {SA<:StaticArray} = SA(Tuple(sa))
+@inline convert(::Type{SA}, sa::StaticArray) where {SA<:Scalar} = SA((sa[],))
```

Comment:
Maybe comment that it's not redundant?
```suggestion
@inline convert(::Type{SA}, sa::StaticArray) where {SA<:Scalar} = SA((sa[],))  # disambiguation
```